overte-builder

```diff
@@ -51,10 +51,18 @@ my $all_targets = {
 	'vhacd-util'          => { service => 0 }
 };
 
+if ( $arch eq "x86_64" ) {
+	my @system_lib_paths = ("/usr/lib64", "/usr/lib/x86_64-linux-gnu", "/usr/lib");
```

Comment:
`my` in Perl declares a locally scoped variable. Doing this inside an if doesn't work because that keeps the scope inside the `if`.

You have to do:

```pl
my @system_lib_paths;

if ( $arch eq "x86_64" ) {
    @system_lib_paths = ("/usr/lib64", "/usr/lib/x86_64-linux-gnu", "/usr/lib");
...
}